site/content/integrate/apps/bindings/_index.md

```diff
@@ -35,24 +35,26 @@ Bindings are organized by top level locations. Top level bindings just need to d
-| `location` | string | Name of this location. The whole path of locations will be added in the context |
-| `icon`     | string | (Optional) Either a fully-qualified URL, or a path for an app's static asset.   |
-| `label`    | string | Text to show in the item.                                                       |
-| `call`     | Call   | Call to perform.                                                                |
+| Name       | Type   | Description                                                                                                       |
+| :--------- | :----- | :---------------------------------------------------------------------------------------------------------------- |
+| `location` | string | Name of this location. The whole path of locations will be added in the context. Must be unique in its level.     |
+| `icon`     | string | (Optional) Either a fully-qualified URL, or a path for an app's static asset.                                     |
+| `label`    | string | (Optional) Text to show in the item. Defaults to location. Must be unique in its level.                           |
+| `call`     | Call   | (Optional) Call to perform. You must provide a Call if there is no Form, or the Form itself does not have a Call. |
+| `form`     | Form   | (Optional) Modal form to open. You must provide a Form with a Call if there is no Call defined in the Binding.    |
 
 The call for these bindings will include in the context the user ID, the post ID, the root post ID if any, the channel ID and the team ID.
 
 ### `/channel_header` bindings
 
-| Name       | Type   | Description                                                                      |
-| :--------- | :----- | :------------------------------------------------------------------------------- |
-| `location` | string | Name of this location. The whole path of locations will be added in the context. |
-| `icon`     | string | (Optional) Either a fully-qualified URL, or a path for an app's static asset.    |
-| `label`    | string | Text to show in the item on mobile and webapp collapsed view.                    |
-| `hint`     | string | Text to show in tooltip.                                                         |
-| `call`     | Call   | Call to perform.                                                                 |
+| Name       | Type   | Description                                                                                                                 |
+| :--------- | :----- | :-------------------------------------------------------------------------------------------------------------------------- |
+| `location` | string | Name of this location. The whole path of locations will be added in the context. Must be unique in its level.               |
+| `icon`     | string | (Optional / Webapp Required) Either a fully-qualified URL, or a path for an app's static asset.                             |
```

Comment:
```suggestion
| `icon`     | string | (Optional/Web App required) Either a fully-qualified URL, or a path for an app's static asset.                             |
```

Reply:
Does this mean it's required in/by the Web App?

Reply:
This means that in general is optional, but it is required in the web app. Since there are only two different agents (webapp or mobile) we could phrase it as "optional in mobile, required in webapp", but I wanted to be "future proof".

Reply:
I thought that might be the case but just wanted to check. I like the idea of future-proofing it! :)